cmd/thanos/receive.go

```diff
@@ -916,6 +918,8 @@ func (rc *receiveConfig) registerFlag(cmd extkingpin.FlagClause) {
 
 	cmd.Flag("tsdb.allow-overlapping-blocks", "Allow overlapping blocks, which in turn enables vertical compaction and vertical query merge. Does not do anything, enabled all the time.").Default("false").BoolVar(&rc.tsdbAllowOverlappingBlocks)
 
+	cmd.Flag("tsdb.max-block-bytes", "Maximum size for local TSDB blocks").Default("0").Int64Var(&rc.tsdbMaxBytes)
```

Comment:
Can we use BytesVar instead? https://github.com/prometheus/prometheus/blob/main/cmd/prometheus/main.go#L327

Also please extend the flag description similar to the one in Prometheus. We probably also need to mention it is per tenant TSDB